teuthology/task/internal/__init__.py

```diff
@@ -459,6 +459,78 @@ def coredump(ctx, config):
                         'Found coredumps on {rem}'.format(rem=rem)
 
 
+import teuthology
+@contextlib.contextmanager
+def archive_artifacts(ctx, config):
+    """
+    Stash a artifacts.
+    override:
+        archive_artifacts:
+        - '/etc/zypp/repos.d'
+        - '/etc/resolv.conf'
+        - name: 'zypper-repos'
+          exec: 'sudo zypper lr'
```

Comment:
`sudo zypper lr -upEP`

Reply:
Well, this is just an example, the idea is that you can specify any command sequence supposed to be executed to capture output in the accompanying yaml file.